adlfs/tests/conftest.py

```diff
@@ -28,7 +28,7 @@ def host(request):
     return request.config.getoption("--host")
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
This had no measurable impact on test runtime, therefore I'm hoping its ok to change this.